.github/workflows/tests.yml

```diff
@@ -21,10 +21,10 @@ jobs: # Docs: <https://git.io/JvxXE>
         setup: [basic, lowest]
         laravel: [default]
         coverage: [yes]
-        php: ['8.0', '8.1', '8.2']
+        php: ['8.1', '8.2', '8.3']
```

Comment:
Laravel 11 does not support php 8.1, so if you want to leave php 8.1 support, you have to specify that you only test it with Laravel 10.

Reply:
Thank you! Added `include` section for php 8.1